scripts/gwf

```diff
@@ -58,6 +61,94 @@ execfile(args.workflow_file)
 from gwf_workflow.workflow import build_workflow, schedule, dependencies
 workflow = build_workflow()
 
+if args.performance:
+
+    if os.path.isfile('.jobs.ids'):
```

Comment:
Useless check, `open()` is going to fail anyway if the file is not a file (?) or it doesn't exist (if this is what you want to check for, don't do it. Just catch the exception raised by `open()`).